packages/client/modules/pages/PageDeletedHeader.tsx

```diff
@@ -52,18 +55,26 @@ export const PageDeletedHeader = (props: Props) => {
   const relativeTime = relativeDate(deletedAt, {smallDiff: 'just now'})
   const {preferredName} = deletedByUser
   return (
-    <div className='flex w-full items-center justify-center bg-tomato-500 font-semibold text-white'>
+    <div className='flex h-10 w-full items-center justify-center bg-tomato-500 font-semibold text-white'>
       <div className='pr-4'>{`${preferredName} moved this page to the trash ${relativeTime}`}</div>
-      <Button
-        variant='outline'
-        onClick={restorePage}
-        className='m-1 text-white hover:bg-tomato-400'
-      >
-        Restore page
-      </Button>
-      <Button variant='outline' onClick={deletePage} className='m-1 text-white hover:bg-tomato-400'>
-        Delete forever
-      </Button>
+      {viewerAccess === 'owner' && (
```

Comment:
should editors be able to archive/delete/restore pages? or just owners? right now it's just owners, and so I fixed the UI to reflect that